tools/charsuites.php

```diff
@@ -198,7 +201,7 @@ function output_characters_slice($slice)
             echo "<span class='gs-char'>$char</span><br>";
             echo "<span class='gs-codepoint'>$codepoint</span>";
             echo "</td>";
-        } else {
+        } else { /** @phpstan-ignore-line */
```

Comment:
This is needed to neuter a PHPStan warning: this `else` is never reached based on the type as `string` can never be `null`.

We can probably remove this branch but for now, I am sticking with disabling it.

Reply:
Maybe I'm not understanding what you mean here? Several of the picker sets do include `null`, and this function is used to display the picker sets.

Reply:
Indeed, I misread the code and mistyped a set as `string[]` (ie not allowing for the possibility of `null` in the array, which made the `else` branch unreachable). Updated several types to `string|null` removes the need for `@phpstan-ignore-line` on this line.

As a side note, do you know what `null` means in a Picker's range? Some kind of placeholder char?

Reply:
IIRC we use null for layout (an empty placeholder).

Reply:
> IIRC we use null for layout (an empty placeholder).

Yes, several of the character suites use null in picker sets.